Sources/SourceKitLSP/SourceKitLSPServer.swift

```diff
@@ -994,6 +994,14 @@ extension SourceKitLSPServer: MessageHandler {
 
     logger.log("Received request \(id): \(params.forLogging)")
 
+    if let textDocumentRequest = params as? any TextDocumentRequest {
+      // When we are requesting information from a document, poke preparation of its target. We don't want to wait for
+      // the preparation to finish because that would cause too big a delay.
+      // In practice, if the user is working on a file, we'll
```

Comment:
Nit: Incomplete comment

Reply:
Oops, it was getting late last night. Finished the comment to

```
      // In practice, while the user is working on a file, we'll get a text document request for it on a regular basis,
      // which prepares the files. For files that are open but aren't being worked on (eg. a different tab), we don't
      // get requests, ensuring that we don't unnecessarily prepare them.
```